draft-ietf-jose-json-proof-token.md

```diff
@@ -128,25 +128,32 @@ The following is an example JWP Protected Header that includes a `cid`:
 
 <{{./fixtures/template/jpt-issuer-protected-header-with-cid.json}}
 
-# Payloads
+# Presented Claims and Proofs
 
-> Editor's Note: This section is incomplete. Use it only as an indicator of the intended direction.
-
-Application resolves each claim as required when processing the JPT.  Resolution can result in one of three things:
-1. A disclosed JSON value
-2. A custom proof method
-3. A `null` value
+Each claim in the issued form of the JPT results in one of three things in the presented form of the JPT:
+1. A disclosed JSON value.
+1. An indicator that the value was not disclosed.
+1. An algorithm-specific proof method.
 
 ## Disclosed
 
-Always an octet string of valid JSON text.
+A disclosed payload is represented as a base64url-encoded UTF-encoded octet string
+representing a valid JSON value.
+
+## Undisclosed
+
+The placeholder indicating that a payload was not disclosed is represented as
+a `null` value for the JWP JSON Serialization
+or the empty string value for the JWP Compact Serialization.
```

Comment:
Recommend we leave this to the core JWP spec. I'm specifically trying to consolidate special cases like this under the serialization rules.

Reply:
I updated the text to reference the representation in JWP rather than repeating it.